source/connect/disconnect.txt

```diff
@@ -11,3 +11,16 @@ Disconnect Connection
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+Disconnecting from a MongoDB instance closes the Compass connection to
+the active instance and returns the Compass view to the initial
+connection dialog.
+
+Disconnect Procedure
```

Comment:
[suggestion]

I'm not sure this heading is necessary. I might replace this heading with something like: "To disconnect from a MongoDB deployment: "

Reply:
Done.